test/suite/test_checkpoint06.py

```diff
@@ -71,16 +80,37 @@ def test_rollback_truncation_in_checkpoint(self):
         start.set_key(5)
         end = self.session.open_cursor(self.uri)
         end.set_key(9995)
+        self.session.breakpoint()
```

Comment:
This seems to be for debugging. Is it still needed?

Reply:
No, removed. Thanks.